Source/FolioReaderFontsMenu.swift

```diff
@@ -243,7 +243,12 @@ class FolioReaderFontsMenu: UIViewController, SMSegmentViewDelegate, UIGestureRe
             scrollDirection = readerConfig.scrollDirection
         }
 
-        layoutDirection.selectSegmentAtIndex(scrollDirection?.rawValue ?? 0)
```

Comment:
Have you tried on PR #237 because I am currently reviewing that and this is not happening, how do I reproduce that?